### PR TITLE
Job attributes

### PR DESCRIPTION
**Pull request by tielur**
What is this?
---
- Adds `add_to_group` to match on printer_attributes to better parse the response
- Adds several CUPS operations

### CODE REVIEW

File: lib/encoder.ex [hunk 1]
```diff
@@ -14,17 +14,59 @@ defmodule Hippy.Encoder do
...
 
   defp encode_attribute({tag, name, value})
-       when tag in [:charset, :uri, :natural_language, :keyword, :name_without_language] do
+       when tag in [
+              :charset,
+              :uri,
+              :natural_language,
+              :keyword,
+              :name_without_language,
+  :text_without_language,
```
Comment (mpichette) [inline lib/encoder.ex]: Might want to just run format on it

File: lib/operation/send_document.ex [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  end
+
+  defp generate_name, do: DateTime.to_iso8601(DateTime.utc_now(), :basic)
+end
+
+defimpl Hippy.Operation, for: Hippy.Operation.SendDocument do
+  def build_request(op) do
+    target = String.replace(op.printer_uri, ~r/^http(s)?/, "ipp")
+
+    #    Hexate.encode(op.document) |> IO.puts()
```
Comment (mpichette) [inline lib/operation/send_document.ex]: Probably should just take this out for cleanliness

File: lib/protocol/binary.ex [hunk 3]
```diff
@@ -1,5 +1,5 @@
 defmodule Hippy.Protocol.Binary do
   defmacro byte do
-    quote do: signed-8
```
Comment (mpichette) [inline lib/protocol/binary.ex:3]: Normally the version without spaces is more common in typedefs

Comment (tielur) [reply lib/protocol/binary.ex:3]: @mpichette the formatter is changing this but I also noticed this isn't even being used, I'm also not sure if this is doing what I think it's suppose to. I think Elixir is interpreting this as the variable `signed` `minus` `8`, is that the intention?

Comment (tielur) [reply lib/protocol/binary.ex:3]: Should I just delete this module maybe?

**Review by mpichette — APPROVED** [review]

**Pull request merged**